src/Symfony/Component/Finder/Tests/FinderTest.php

```diff
@@ -339,6 +339,10 @@ public function testInWithNonDirectoryGlob()
 
     public function testInWithGlobBrace()
     {
+        if (!\defined('GLOB_BRACE')) {
```

Comment:
Should we also add such a test in the code to alert the developer of the issue?

Reply:
I'm not sure we can reliably implement this without a glob parser. We already have a glob parser, but not for this purpose (see `Glob::toRegex()`)

Reply:
Fyi, Zend has implemented a fallback in it's stdlib: https://github.com/zendframework/zend-stdlib/blob/master/src/Glob.php#L89-L168